chromedp.go

```diff
@@ -385,6 +388,60 @@ func (c *Context) attachTarget(ctx context.Context, targetID target.ID) error {
 	return nil
 }
 
+// Authentication proxy
+func Authentication(login string, password string) Tasks {
+	return Tasks{
+		fetch.Enable().WithPatterns([]*fetch.RequestPattern{{"*", "", ""}}).WithHandleAuthRequests(true),
+		ActionFunc(func(ctx context.Context) error {
+			if login != "" && password != "" {
+				c := FromContext(ctx)
+				if c == nil || c.Allocator == nil || c.cancel == nil {
+					return ErrInvalidContext
+				}
+				id := 1000
```

Comment:
This ID looks like a magic number. Maybe it will be useful to have some comment that describes this variable, or even move this to constants?